src/ui/SmartSnippet/SmartSnippet.ts

```diff
@@ -301,19 +301,21 @@ export class SmartSnippet extends Component {
   }
 
   private renderSourceTitle() {
-    return this.buildLink(Utils.getFieldValue(this.lastRenderedResult, <string>this.options.titleField), SOURCE_TITLE_CLASSNAME);
+    const link = this.buildLink();
+    link.innerText = Utils.getFieldValue(this.lastRenderedResult, <string>this.options.titleField);
+    link.classList.add(SOURCE_TITLE_CLASSNAME);
+    return link;
   }
 
   private renderSourceUrl() {
-    const uri = this.options.hrefTemplate
-      ? StringUtils.buildStringTemplateFromResult(this.options.hrefTemplate, this.lastRenderedResult)
-      : this.lastRenderedResult.clickUri;
-    return this.buildLink(uri, SOURCE_URL_CLASSNAME);
+    const link = this.buildLink();
+    link.innerText = link.href;
```

Comment:
The href, even on IE11, contains the full URL even if it was originally set to a relative URL.